cloudify_azure/tests/resources/test_base.py

```diff
@@ -44,7 +44,11 @@ def setUp(self):
                 'tenant_id': self.tenant_id,
                 'client_id': '1234567',
                 'client_secret': '1234-1234-1234-1234',
-                'subscription_id': self.subscription_id
+                'subscription_id': self.subscription_id,
+                'endpoint_resource': constants.OAUTH2_MGMT_RESOURCE,
+                'endpoint_verify': True,
+                'endpoints_resource_manager': constants.CONN_API_ENDPOINT,
+                'endpoints_active_directory': constants.OAUTH2_ENDPOINT
```

Comment:
Are these parameters ignored if the user does not provide them?

Reply:
Yes, default values is provided in plugins.yaml.